src/Arcus.EventGrid.Testing/Infrastructure/Hosts/EventConsumerHost.cs

```diff
@@ -13,8 +13,13 @@ namespace Arcus.EventGrid.Testing.Infrastructure.Hosts
     /// </summary>
     public class EventConsumerHost
     {
-        private static readonly Dictionary<string, string> _receivedEvents = new Dictionary<string, string>();
-        protected readonly ILogger _logger;
+        // TODO: shouldn't this be a 'ConcurrentDictionary'?
+        private static readonly Dictionary<string, string> ReceivedEvents = new Dictionary<string, string>();
+
+        /// <summary>
+        ///     Gets the logger associated with this event consumer.
+        /// </summary>
+        protected ILogger Logger { get; }
```

Comment:
Curious - Why are we exposing this?

Reply:
IDK, it was a `protected` field; so I changed it to a `protected` property. I guess it's because some childs use it to log specific messages (like the _Service Bus_ implementation)

Reply:
Ok, I didn't see that it was protected so that's ok for me.